Follower scroll

pahumadad opened the pull request:
Implement a method to update the scroll position to follow the cursor

fortizc committed:
Initial idea of how the scroll issue would be fixed

pahumadad committed:
Update the method to move the scroll position

fortizc commented on blackboard/gui/mainwin.py:
```diff
@@ -58,6 +58,23 @@ def __add_txt_view(self):
+        if cursor_pos >= view_start and cursor_pos < view_end:
+            return
+        if cursor_pos >= view_end:
+            adj.set_value(view_start + cursor_height)
```
Bug here! in OSX cursor lost below of the textview

pahumadad replied:
Fixed in e4ed7c9

pahumadad committed:
Fix the cursor lost at the end at the text view

When the cursor size is not multiple of view size the cursor
appears with lost because in the last line it hasn't enough space
(pixels). To avoid that, the modulus between the cursor size and
the view size must be subtracted from the view size.
